src/style.css

```diff
@@ -1,3 +1,73 @@
+@media (min-width: 1024px), (max-width: 1200px) {
+  header {
+    min-width: 924px;
+    max-width: 1100px;
+  }
+}
+
+html {
+  --textcolor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+  padding: 0 50px;
+}
+
+.home_page {
+  height: 40px;
+}
+
+.box {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-family: "Roboto", "SansSerifCollection", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding: 23px 0; instead